runtime/development/src/lib.rs

```diff
@@ -283,27 +290,68 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 parameter_types! {
 	pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT.saturating_div(4);
 	pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT.saturating_div(4);
+	pub const RelayOrigin: AggregateMessageOrigin = AggregateMessageOrigin::Parent;
 }
 
 impl cumulus_pallet_parachain_system::Config for Runtime {
 	// Using AnyRelayNumber only for the development & demo environments,
 	// to be able to recover quickly from a relay chains issue
 	type CheckAssociatedRelayNumber = cumulus_pallet_parachain_system::AnyRelayNumber;
-	type DmpMessageHandler = DmpQueue;
+	type WeightInfo = (); //TODO-1.7 check weights
```

Comment:
Now `cumulus_pallet_parachain_system` has been added to the benchmarking list so we ideally need to point to the autogenerated weights.

Reply:
Using the defensive weights